conf/airframes/ENAC/quadrotor/anton_indi_aruco.xml

```diff
@@ -11,6 +11,8 @@
   </description>
 
   <firmware name="rotorcraft">
+    <!--autopilot name="rotorcraft_autopilot"/-->
```

Comment:
why?

Reply:
It was for testing purpose in simulation, but not validated in real flight yet